test/haxesharp/collections/DictionaryTest.hx

```diff
@@ -62,15 +62,7 @@ class DictionaryTest
         var testKeyNotFoundDictionary = new Dictionary<Int,Int>();
         testKeyNotFoundDictionary.set(1, 1);
 
-        try
-        {
-            testKeyNotFoundDictionary.get(2);
-            Assert.fail("testKeyNotFoundDictionary.get(2) did not throw!");
-        }
-        catch (e:KeyNotFoundException)
-        {
-            //Pass
-        }
+        Assert.throws(KeyNotFoundException, (_) => testKeyNotFoundDictionary.get(2));
```

Comment:
I was wondering about this. Does this work with `() =>` instead of `(_)`? What exactly does the underscore represent?

Reply:
With hxslam, it does not: https://github.com/bynuff/hxslam#usage

Reply:
That explains why my other tests weren't compiling. Thanks for figuring this out.